package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "simple-progress-webpack-plugin",
 	"description": "A simple progress plugin for Webpack, coming with four different logging output formats.",
-	"version": "1.1.1",
+	"version": "1.1.2",
```

Comment:
Please don't increment version numbers in PRs, the automatic release process will do it for us :)